lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -78,10 +78,21 @@ def instructor_dashboard_2(request, course_id):
         _section_membership(course, access),
         _section_student_admin(course, access),
         _section_data_download(course, access),
-        _section_analytics(course, access),
     ]
 
-    #check if there is corresponding entry in the CourseMode Table related to the Instructor Dashboard course
+    analytics_dashboard_message = None
+    if settings.ANALYTICS_DASHBOARD_URL:
```

Comment:
Moved this clause up so that the analytics section would still appear in the proper position in the `sections` list.